Library/Homebrew/formula.rb

```diff
@@ -1614,6 +1614,10 @@ def opt_or_installed_prefix_keg
     end
   end
 
+  def opt_or_installed_version
```

Comment:
```suggestion
  def opt_linked_version
```

or something, maybe?

Reply:
Perhaps `preferred_version` or `available_version`?

Reply:
Could even just be `installed_version`?

Reply:
What about `opt_or_installed_prefix_keg`? We can't rename it to `installed_prefix` since that's already taken, and `installed_prefix_keg` would be confusing since `installed_prefix` already exists

Reply:
It conflicts with the `installed_version` method that already exists 😢